match/README.md

```diff
@@ -174,20 +174,17 @@ match --help
 
 #### Handle multiple targets
 
-If you have several targets with different bundle identifiers, call `match` for each of them:
+If you have several targets with different bundle identifiers, supply them as a comma-separated list to match:
 
 ```
-match appstore -a tools.fastlane.app
-match appstore -a tools.fastlane.app.watchkitapp
+match appstore -a tools.fastlane.app,tools.fastlane.app.watchkitapp
 ```
 
 You can make this even easier using [fastlane](https://github.com/fastlane/fastlane/tree/master/fastlane) by creating a match lane like this:
 
 ```
 lane :match do
```

Comment:
Ah, the lane shouldn't be called match actually, I submitted a separate PR to change that https://github.com/fastlane/fastlane/pull/6074, make sure to rebase after it got merged